deployment/data-streams/changeset/types/types.go

```diff
@@ -22,3 +24,25 @@ type (
 		OverrideRoot bool
 	}
 )
+
+type OwnershipFeature struct {
+	Transfer           bool // If true, MCMS takes ownership
+	DeployMCMS         bool
```

Comment:
I like using verbs for bools, like ShouldTransfer, ShouldDeploy, etc.

Reply:
sounds good - fixed